SimPEG/DataMisfit.py

```diff
@@ -1,11 +1,14 @@
+from __future__ import print_function
 import numpy as np
 import scipy.sparse as sp
 import gc
-import Utils
-import Survey
-import Problem
+from SimPEG import Utils
+from SimPEG import Survey
+from SimPEG import Problem
 
+from six import add_metaclass
 
+@add_metaclass(Utils.SimPEGMetaClass)
```

Comment:
Both of them worked, I just thought that the decorator form using <add_metaclass> was a little clearer than the <with_metaclass>, because it explicitly states which thing you're adding is the metaclass.
Also the codecy check likes it better.